tests/projects/service/deployment/project.yml

```diff
@@ -13,6 +13,9 @@ build:
         test: "Test"
         acceptance: "Acceptance"
         production: "Production"
+    credentials:
+      - key: SOME_CREDENTIAL
+        id: SOME_CREDENTIAL
```

Comment:
is there no test where we can set an env value and check if it actually shows up?

Reply:
I was trying to think, but I don't think so.. With my changes to throw in case the env var is not set, at least the IT tests should fail